src/assets/css/main.css

```diff
@@ -88,6 +88,15 @@
   color: var(--bs-gray) !important;
 }
 
+input[type="file"]::file-selector-button {
```

Comment:
Does bootstrap 5 not have a a11y-compatible way to file upload buttons? Not sure why this custom css is necessary.

Reply:
There is this: https://getbootstrap.com/docs/5.2/forms/input-group/#custom-file-input

Reply:
Ah, of course! Thank you. I've just updated this PR to use those classes.